src/sentry/static/sentry/app/views/onboarding/projectSetup/firstEventIndicator.jsx

```diff
@@ -0,0 +1,140 @@
+import {keyframes} from 'emotion';
+import PropTypes from 'prop-types';
+import React from 'react';
+import posed, {PoseGroup} from 'react-pose';
+import styled from 'react-emotion';
+
+import {t} from 'app/locale';
+import Button from 'app/components/button';
+import EventWaiter from 'app/views/onboarding/projectSetup/eventWaiter';
+import InlineSvg from 'app/components/inlineSvg';
+import space from 'app/styles/space';
+
+const FirstEventIndicator = props => {
+  const {orgId, projectId} = props;
```

Comment:
~~nit: you could destructure from function args to save some chars~~

Idk what's intended here since `<EventWaiter>` gets explicit props and `<Waiting>` and `<Success>` don't.

Reply:
5ca1a82 Moved things around a bit.